lib/access_check_filter.js

```diff
@@ -64,6 +66,9 @@ module.exports = (hyper, req, next, options, specInfo) => {
                 }
 
                 return contentPromise.then((content) => redirect(content, location, options));
+            } else if (content.status === 302) {
```

Comment:
I see -- that's pretty subtle. Do you think an explaining comment here could be helpful to remind ourselves about how this interacts with parsoid.js?